src/screens/NewsScreen/RenderItem.js

```diff
@@ -79,7 +79,7 @@ const styles = StyleSheet.create({
     borderRadius: SIZES.radius,
     backgroundColor: COLORS.white,
     elevation: 0.0,
-    borderColor: COLORS.gray,
+    borderColor: COLORS.blackgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.gray` to `COLORS.blackgrey` seems fine as long as the `COLORS.blackgrey` is defined in the `COLORS` object and it fits the overall color scheme of the application. Please ensure that this color change is consistent with the design guidelines of your application.
```diff
-    borderColor: COLORS.gray,
+    borderColor: COLORS.blackgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->